jcm/model.py

```diff
@@ -186,10 +186,30 @@ def conserve_global_mean_surface_pressure(u, u_next):
                 log_surface_pressure=u_next.log_surface_pressure.at[0, 0, 0].set(u.log_surface_pressure[0, 0, 0])
             )
         
+        # create diffusion filter function handles
+        diffuse_div = self._make_diffusion_fn(
+            self.diffusion.div_timescale,
+            self.diffusion.div_order,
```

Comment:
Sorry, do we need diffusion to be on `self` if we're only using it here in the constructor?